build.gradle.kts

```diff
@@ -172,8 +172,8 @@ allprojects {
       param.environment("PROJECT_VERSION", project.version)
 
       // Gravitino CI Docker image
-      param.environment("GRAVITINO_CI_HIVE_DOCKER_IMAGE", "apache/gravitino-ci:hive-0.1.19")
-      param.environment("GRAVITINO_CI_KERBEROS_HIVE_DOCKER_IMAGE", "apache/gravitino-ci:kerberos-hive-0.1.5")
+      param.environment("GRAVITINO_CI_HIVE_DOCKER_IMAGE", "unknowntpo/gravitino-ci:hive-change-hive-username")
+      param.environment("GRAVITINO_CI_KERBEROS_HIVE_DOCKER_IMAGE", "unknowntpo/gravitino-ci:kerberos-hive-change-hive-username")
```

Comment:
@unknowntpo I have already polished the Docker Image. Please modify these Docker image versions, and let us to run ITs. Thanks!

Reply:
@xunliu Done.